cylc/flow/scheduler.py

```diff
@@ -1643,7 +1644,10 @@ async def _shutdown(self, reason: Exception) -> None:
                 self.resume_workflow(quiet=True)
         elif isinstance(reason, SchedulerError):
             LOG.error(f'Workflow shutting down - {reason}')
-        elif isinstance(reason, CylcError):
+        elif (
+            isinstance(reason, CylcError)
+            or isinstance(reason, TemplateVarLanguageClash)
```

Comment:
I suspect that you are right, but I'd like @oliver-sanders  to weigh in on this one...

Reply:
I don't think a `ParsecError` is an "expected error" during workflow runtime in that configuration issues should have been handled before we reach this point.

Can we push this error into `cylc.flow.config` so that it fails `cylc validate`?

Reply:
It did fail on validate already
but it also fails on play with traceback - that was the whole point of this PR>.

Reply:
What am I missing to reproduce the validation failure? I've tried creating a `flow.cylc` with `#!empy` shebang, and a `rose-suite.conf` with this
```ini
[jinja2:suite.rc]
MEOW=4
```
but it passes `cylc validate` without error

Reply:
Ah yes, other `ParsecError`s also get traceback on `cylc play` (but no traceback on `cylc validate`), so I go back to my prior opinion that `ParsecError`s should be "expected"

Reply:
If a `ParsecError` occurs during the loading of the config, that's an "expected error", if it occurs at a later time that is not.

Took a quick look at the scheduler, any `ParsecError`s raised in `cylc.flow.config` end up handled the same way as a `CylcError` raised at runtime which is somewhat unhelpful. So perhaps just add it for the mo, i would like to come back to this sometime as reloading a broken config will currently pull down the scheduler.

Reply:
https://github.com/cylc/cylc-flow/issues

Reply:
#4216?